src/lang/extra/czech.txt

```diff
@@ -572,81 +1106,1083 @@ STR_PROGSIG_INSERT                                              :Vložit
+STR_BUY_VEHICLE_TRAIN_BUY_REFIT_WAGON_TOOLTIP                   :{BLACK}Koupit a přestavět zvýrazněné tažené vozidlo. Shift+kliknutí zobrazí (bez nákupu) odhad ceny
+STR_BUY_VEHICLE_TRAIN_RENAME_LOCOMOTIVE_BUTTON                  :{BLACK}Přejmenovat hnací vozidlo
+STR_BUY_VEHICLE_TRAIN_RENAME_WAGON_BUTTON                       :{BLACK}Přejmenovat vůz
+STR_BUY_VEHICLE_TRAIN_RENAME_LOCOMOTIVE_TOOLTIP                 :{BLACK}Přejmenovat lokomotivu či hnací vůz nebo jednotku
+STR_BUY_VEHICLE_TRAIN_RENAME_WAGON_TOOLTIP                      :{BLACK}Přejmenovat tažené vozidlo
+
+STR_QUERY_RENAME_TRAIN_TYPE_LOCOMOTIVE_CAPTION                  :{WHITE}Přejmenovat tuto řadu lokomotiv
+STR_QUERY_RENAME_TRAIN_TYPE_WAGON_CAPTION                       :{WHITE}Přejmenovat tuto řadu vozů
+
+STR_VEHICLE_VIEW_SEND_TO_DEPOT_TOOLTIP_SHIFT                    :{STRING}. Shift+kliknutím vyber, do kterého
+STR_VEHICLE_VIEW_SEND_TO_DEPOT_MENU                             :{BLACK}Zrušit odeslání vozidla do depa/hangáru. Ctrl+kliknutí otevře nabídku. Shift+kliknutí pro výběr depa/hangáru
+
+STR_VEHICLE_VIEW_TRAIN_CENTER_TOOLTIP_EXTRA                     :{STRING}{BLACK}{}Shift+kliknutí zapne vykreslování trasy vozidla, i když není jeho okno vybráno
+
+STR_VEHICLE_STATUS_LOADING_UNLOADING_ADVANCE                    :{STRING}, {VELOCITY}
+STR_VEHICLE_STATUS_BROKEN_DOWN_VEL                              :{RED}Porucha - {STRING}, {LTBLUE} {VELOCITY}
+STR_VEHICLE_STATUS_BROKEN_DOWN_VEL_SHORT                        :{RED}Porucha - {STRING}
+STR_VEHICLE_STATUS_TRAIN_STUCK_WAIT_RESTRICTION                 :{ORANGE}Čeká kvůli omezení směrování
+STR_VEHICLE_STATUS_TRAIN_WAITING_TIMETABLE                      :{ORANGE}Čeká podle jízdního řádu
+STR_VEHICLE_STATUS_TRAIN_REVERSING                              :{ORANGE}Obrací se, {VELOCITY}
+STR_VEHICLE_STATUS_TRAIN_MOVING_DEPOT                           :{ORANGE}Posunuje v depu
+
+STR_BREAKDOWN_TYPE_CRITICAL                                     :selhání agregátů
+STR_BREAKDOWN_TYPE_EM_STOP                                      :nouzové zastavení
+STR_BREAKDOWN_TYPE_LOW_SPEED                                    :omezení na {VELOCITY}
+STR_BREAKDOWN_TYPE_LOW_POWER                                    :{COMMA}% výkon
+STR_BREAKDOWN_TYPE_HIT_RV                                       :srážka se silničním vozidlem
+STR_BREAKDOWN_TYPE_BRAKES_OVERHEATED                            :přehřáté brzdy
+STR_BREAKDOWN_TYPE_DEPOT                                        :Míří do hangáru na {STATION} pro provedení oprav
+STR_BREAKDOWN_TYPE_LANDING                                      :Míří na {STATION} pro nouzové přistání
+
+STR_ERROR_TRAIN_TOO_HEAVY                                       :{WHITE}Souprava {VEHICLE} je příliš těžká
+
+STR_VEHICLE_STATUS_HEADING_FOR_DEPOT_SELL_VEL                   :{CREAM}Míří na {PUSH_COLOUR}{RED}prodej{POP_COLOUR} do {DEPOT}, {VELOCITY}# Cargo type orders Window
```

Comment:
It seems there is some stray text here

Reply:
Oh, yes, sorry. In addition, there was a wrong translation of the next string. Both fixed